### ISSUE TITLE
Projektabschluss

### ISSUE THREAD

**Issue by JirkaDellOro** [JirkaDellOro/Computergrafik.Online#256]
Grundsätzliches, dass in der Folge noch verbessert werden soll (nicht unbedingt vom Team des Projektstudiums)

- [ ] Der Infotext ist oft noch sehr spärlich und schlecht gesetzt
- [ ] Fonts in den Startbildern der Videos sind nicht einheitlich

**Comment by droda1** [comment]
habe Kapitel 1 umbenannt und den Informationstext der Landingpage überarbeitet. Passt der Text nun so oder benötigen wir mehr Bedienungshinweise?

**Comment by droda1** [comment]
@KevinBlumtritt @HendrikLange 
könnt ihr vielleicht mal eine Liste der videos erstellen, auf die "Fonts in den Startbildern der Videos sind nicht einheitlich"
zutrifft?

**Comment by JirkaDellOro** [comment]
Ihr könnt selbst entscheiden, ob ihr die Punkte angehen wollt. Ich bewerte den jetzt aktuellen Stand. Als Icon kann ich mir eine Hand gut vorstellen.

**Comment by droda1** [comment]
Alles kar, ist das in etwa das, was sie sich vorgestellt haben?
https://icon-icons.com/de/symbol/ganze-Hand-Hand/4171

**Comment by JirkaDellOro** [comment]
Ich denke, dass es etwas abstrahierter und monochrom besser in die Anwendung passt.

**Comment by JirkaDellOro** [comment]
Ich bin nun mit der Durchsicht fertig. Nun muss ich zur Zensur schreiten und hierzu hätte ich gerne eine Abschlussbesprechung mit euch. Kann jemand am Montag um 9.00 Uhr nicht? Ansonsten gerne auch am Sonntag...

**Comment by droda1** [comment]
Montag passt für mich.

**Comment by Ahornzweig** [comment]
Für mich auch.

**Comment by HendrikLange** [comment]
Passt.

**Comment by StfnMerdi** [comment]
passt für mich auch

**Comment by xcluu** [comment]
Für mich auch.

**Comment by KevinBlumtritt** [comment]
Ist für mich auch in Ordnung.

**Comment by droda1** [comment]
@JirkaDellOro bei mir steht jetzt für Teil 1 und Teil 2 eine 4,0 drinnen, ist das korrekt so?

**Comment by JirkaDellOro** [comment]
Nein, ich habe außer "bestanden" (4.0) noch keine Noten vergeben. Ist die Statusliste fertig?

**Comment by Ahornzweig** [comment]
Ja, die sollte fertig sein.

**Comment by Ahornzweig** [comment]
Es fehlen jedoch noch die Namen zur genauen Zuordnung, da in manchen Kapiteln mehrere Leute gearbeitet haben.

**Comment by droda1** [comment]
@JirkaDellOro wann kann man denn mit einer Note rechnen? Das neue Semester beginnt bald.

**Comment by JirkaDellOro** [comment]
Sollte in den nächsten Tagen eingetragen sein, ich habe sie übermittelt.

**Comment by droda1** [comment]
@JirkaDellOro Vielen Dank!
